tools/Scripts/MakeInstaller.py

```diff
@@ -129,15 +129,15 @@ def installerConfigXml():
                 #'CreateLocalRepository': 'true',
                 #'SaveDefaultRepositories': 'false',
                 #'RepositorySettingsPageVisible': 'false',
-                'RemoteRepositories': {
-                    'Repository': [
-                        {
-                            'Url': onlineRepositoryUrl(),
-                            'DisplayName': f'{CONFIG.app_name} {CONFIG.setup_os}_{CONFIG.setup_arch} repository',
-                            'Enabled': 1,
-                        }
-                    ]
-                },
+                # 'RemoteRepositories': {
+                #     'Repository': [
+                #         {
+                #             'Url': onlineRepositoryUrl(),
+                #             'DisplayName': f'{CONFIG.app_name} {CONFIG.setup_os}_{CONFIG.setup_arch} repository',
+                #             'Enabled': 1,
+                #         }
+                #     ]
+                # },
                 'MaintenanceToolName': maintenance_tool_name,
```

Comment:
is this Diffraction only? Maybe just delete this block if not necessary

Reply:
I will leave it for now as a reminder that we need to get to a "common" set of tools for building / packaging EasyApplications